lib/text.js

```
@@ -297,15 +297,21 @@ exports.SUBSTITUTE = function(text, old_text, new_text, occurrence) {
   } else if (occurrence === undefined) {
     return text.replace(new RegExp(old_text, 'g'), new_text);
   } else {
+    // try to coerce `occurrence` to a positive integer
```

Instead of  a comment I would use a more descriptive test (like we started in ./tests/text.js > VALUE:

`it('should throw an error if occurence is not a positive number')`